README.md

```diff
@@ -73,6 +73,18 @@ Manager parameter store](https://docs.aws.amazon.com/systems-manager/latest/user
   Guacamole
 - `/guacamole/postgres_password`: The non-admin postgres password used by
   Guacamole
+- `/rdp/username`: The username for Guacamole to use when connecting to an
+  instance via RDP
+- `/rdp/password`: The password for Guacamole to use when connecting to an
+  instance via RDP
+- `/vnc/ssh/rsa_private_key`: The private SSH key to use for SFTP file transfer
+  in Guacamole
+- `/vnc/username`: The username for Guacamole to use when connecting to an
+  instance via VNC
+- `/vnc/password`: The password for Guacamole to use when connecting to an
+  instance via VNC
```

Comment:
Another great catch on your part.  Thanks for cleaning up the wreckage I keep leaving in my wake.